.github/workflows/cluster-provider-test-pull-request.yaml

```diff
@@ -152,45 +152,47 @@ jobs:
     - name: Commit and push changes
       if: env.cluster_chart_version_updated == 'true'
       run: |
+        set -u
         cd cluster-aws
         git add helm/cluster-aws/Chart.yaml
         git add helm/cluster-aws/Chart.lock
-        git commit -m "Update cluster chart version to ${{ env.custom_cluster_chart_new_version }}"
+        git commit -m "Update cluster chart version to ${custom_cluster_chart_new_version}"
```

Comment:
Where is `custom_cluster_chart_new_version` coming from? I cannot see it being set in the `env` section.

Reply:
Ah, never mind. You're putting it into `$GITHUB_ENV`. I never used this and instead stored values in outputs which I then used in the `env` object of my steps.